CLI.md.sh

```diff
@@ -193,6 +193,24 @@ generate() {
     command "Environment.List" "env list"
     command "Environment.Remove" "env remove"
     command "Environment.Reset" "env reset"
+
+    group "Address" "address"
+    command "Address.New" "address new"
+    command "Address.List" "address list"
+    command "Address.Dump" "address dump"
+    command "Address.Get" "address get"
+    command "Address.Set" "address set"
+    command "Address.Set" "address delete"
+    command "Address.Switch" "address switch"
+    command "Address.Remove" "address remove"
+    command "Address.Reset" "address reset"
+
+    group "Get" "get"
+    command "Get.Account" "get account"
+    command "Get.Keys" "get keys"
```

Comment:
I would change "keys" to "storage" to keep it consistent with the other "storage-entry" term and docs, otherwise it's confusing.

Reply:
done. will re-generate CLI.md before releasing the new version.